apps/core/src/hooks/useTransactionSummary.ts

```diff
@@ -8,42 +8,41 @@ import {
     getExecutionStatusType,
     getTransactionDigest,
     getTransactionSender,
+    SuiObjectChange,
 } from '@mysten/sui.js';
 import { useMemo } from 'react';
 
 import { getBalanceChangeSummary } from '../utils/transaction/getBalanceChangeSummary';
-import { getObjectChangeSummary } from '../utils/transaction/getObjectChangeSummary';
+import {
+    SuiObjectChangeWithDisplay,
+    getObjectChangeSummary,
+} from '../utils/transaction/getObjectChangeSummary';
 import { getLabel } from '../utils/transaction/getLabel';
 import { getGasSummary } from '../utils/transaction/getGasSummary';
+import { useMultiGetObjectsDisplay } from './useMultiGetObjectsDisplay';
 
-const getSummary = (
```

Comment:
this diff is a little confusing but I just moved the `getSummary` body into the `useTransactionSummary` hook